Add plausible (#857)
publiccodenet/about: activities/tool-management/plausible-analytics.md

---
type: resource
---

# Plausible Analytics

> Plausible Analytics is a simple, lightweight (< 1 KB), open-source and privacy-friendly alternative to Google Analytics. It doesn’t use cookies and is fully compliant with GDPR, CCPA and PECR. The Plausible team are dedicated to making web analytics more privacy-friendly. Their mission is to reduce corporate surveillance by providing an alternative web analytics tool which doesn’t come from the AdTech world.

Reviewer: ```suggestion
Plausible Analytics is a simple, lightweight (< 1 KB), open source and privacy-friendly alternative to Google Analytics. It doesn’t use cookies. Plausible is designed as an alternative to AdTech.

We're currently trialling Plausible (see [issue 64](https://github.com/publiccodenet/jekyll-theme/issues/64).)
```

## Using Plausible Analytics

> Plausible Analytics is integrated into our Jekyll-themed website built with GitHub Pages. With Plausible activated, we have access to a simple open source analytics dashboard. This dashboard includes website metrics such as the number of unique visitors, the number of page views on individual pages and the referral sources which have sent traffic to our site. We will use this data to assess the performance of our site and to figure out what we can do to improve our future efforts.

Reviewer: ```suggestion
Plausible Analytics is integrated into our [Jekyll-theme](https://github.com/publiccodenet/jekyll-theme), which is used by all our websites (since all are built with Github Pages). It gives us a simple open source analytics dashboard, including website metrics such as the number of unique visitors, the number of page views on individual pages and the referral sources that have sent traffic to our site. 

We will use this data to assess the performance of our site and to figure out what we can do to improve our future efforts.
```

## Plausible management

> Plausible Analytics works by loading a script on our website, like Google Analytics. The script is 45x smaller, which makes our website quicker to load. The dashboard can be accessed by the Codebase Stewards and Communication Coordinator.

Reviewer: ```suggestion
Plausible Analytics works by loading a Javascript script on the top navigation of our websites. The dashboard can be accessed by the Codebase Stewards and Communication Coordinator.
```

## Privacy and GDPR

> Plausible: GDPR, CCPA and cookie law compliant site analytics. Read more [in the data policy.](https://plausible.io/data-policy)

Reviewer: ```suggestion
Plausible: GDPR, CCPA and cookie law compliant site analytics. Read more [in the data policy](https://plausible.io/data-policy).
```

> > "We do not track people across their devices and across websites and apps that they visit. All the data is isolated to a single day, single website and single device only. There is no way to know whether the same person visits a site from more than one device or visits another website. When you use our service to measure your website stats, Plausible Analytics will collect information about your visitors. You entrust us with your site data and we take that trust to heart. The privacy of your site data — and it is your data, not ours! — is a big deal to us. By using Plausible, you keep 100% ownership of your website data. Although when using our hosted service, your site analytics are stored on our server in the cloud, you remain completely in control of your site data and you fully own all of your data too. You own all right, title, and interest to your website data. We obtain no rights from you to your website data. We will never sell or share your site data to any third-parties."

Reviewer: ```suggestion
> 
```

---
type: resource
---

# Plausible Analytics

Plausible Analytics is a simple, lightweight (< 1 KB), open source and privacy-friendly analytics tool. It doesn’t use cookies.

> We're currently trialling Plausible (see [issue 64](https://github.com/publiccodenet/jekyll-theme/issues/64).)

Reviewer: ```suggestion
We're currently trialling Plausible (see [issue 64](https://github.com/publiccodenet/jekyll-theme/issues/64)). [See our Plausible dashboard](https://plausible.io/publiccode.net).
```

## Using Plausible Analytics

Plausible Analytics is integrated into our [Jekyll-theme](https://github.com/publiccodenet/jekyll-theme), which is used by all our websites (since all are built with Github Pages). It gives us a simple open source analytics dashboard, including website metrics such as the number of unique visitors, the number of page views on individual pages and the referral sources that have sent traffic to our site.

We will use this data to assess the performance of our site and to figure out what we can do to improve our future efforts.

## Plausible management

> Plausible Analytics works by loading a Javascript script on the top navigation of our websites. The dashboard can be accessed by the Codebase Stewards and Communication Coordinator.

Reviewer: ```suggestion
Plausible Analytics works by loading a Javascript script on the top navigation of our websites. [Our Plausible dashboard is publicly shared]. It can be configured by the Codebase Stewards and Communication Coordinator.
```
